### PR TITLE
Ports pan-slavic from Baystation

### PR DESCRIPTION
**Pull request by Novacat**

### CODE REVIEW

**Comment by Wickedtemp** [comment]
So.. a new language?

**Comment by LSommers** [comment]
> So.. a new language designed to catch Ivan edgelords! Neat!

Can you don't?

**Comment by Wickedtemp** [comment]
..was a reference for all of the troll characters we used to get. "Stereotypical russian assistants drinking themselves to death in the bar with vodka" used to be a semi-regular thing. One in particular was named Ivan. 

"Ivan Lowaski yells, "Ivan does drink lots vodka, that is why his kidney is gone, Da!"
Ivan Lowaski says, "Vodka is Ivans blood"

Was an actual exchange I had with one of them, so.. was referencing these types of trolls. Apologies if that didn't come off as the intention and if I sounded rude D:

**Comment by LSommers** [comment]
> ..was a reference for all of the troll characters we used to get. "Stereotypical russian assistants drinking themselves to death in the bar with vodka" used to be a semi-regular thing. One in particular was named Ivan.

Ah, fair enough. Misunderstood it as a dig towards Slavic or Slavic-inspired characters. I guess I missed that particular flavor of troll, seems memorable. Though I don't think that someone with that low effort of a troll character would think to look through the languages.

**Comment by SpadesNeil** [comment]
Pan Slavic is broken. If you have the language you can't understand it.

**Comment by SpadesNeil** [comment]
https://github.com/VOREStation/VOREStation/issues/6540

File: code/modules/mob/language/station_vr.dm [hunk 1]
```diff
@@ -83,6 +83,19 @@
     "uhk","zir","sc'orth","sc'er","thc'yek","th'zirk","th'esk","k'ayek","ka'mil","sc'","ik'yir","yol","kig","k'zit","'","'","zrk","krg","isk'yet","na'k",
     "sc'azz","th'sc","nil","n'ahk","sc'yeth","aur'sk","iy'it","azzg","a'","i'","o'","u'","a","i","o","u","zz","kr","ak","nrk","tzzk","bz","xic'","k'lax'","histh")
 
+/datum/language/human/slavic
```
Comment (SpadesNeil) [inline code/modules/mob/language/station_vr.dm]: This looks like the source of the problem.

**Pull request merged**